stories/topics/ref-saas-planning-configs.adoc

```diff
@@ -0,0 +1,5 @@
+[id="ref-saas-planning-configs"]
+= Planning the required configurations for {SaaSonAWSShort}
+After accessing your {PlatformNameShort} instance, you can confirm that there is only a control node, and mesh ingress hop nodes configured on the *Instances* page or *Topology* page. 
+Before you can start automation jobs you must add remote execution nodes to the cluster, or connect an OpenShift cluster as a container group. 
+For help with configuring execution nodes see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_automation_mesh_guide_for_vm-based_installations/index#assembly-planning-mesh[Red Hat Ansible Automation Platform Automation Mesh Guide for VM-based installations]
```

Comment:
If you're creating instances (e.g. execution nodes are you creating them entirely using inventory files (i.e a VM-based install) or are you creating them using Controller (an operator-based install), or both. I think you need to distinguish which one you're using.

Reply:
I have asked Scott after your comment from Friday. Just waiting on his response.

Reply:
Scott confirmed:
"The operator-based installation guide, not the VM-based, will be the appropriate link.

The whole guide will be the context...the customer will have to follow the entire process to setup the automation mesh."